src/test/java/web/component/Table.java

```diff
@@ -26,10 +26,24 @@ private List<WebElement> getRows() {
         return rows;
     }
 
+    private List<WebElement> getRowsOnliner() {
```

Comment:
you already have method getRows() in the Table component, there should be one common method getRows() for the table component